kernel/mempool.c

```diff
@@ -0,0 +1,404 @@
+#include <kernel.h>
```

Comment:
it's fine, that;s what "SPDX-License-Identifier: Apache-2.0" does nowadays, we removed all the copyright boilerplate kernel-wide.